### PR TITLE
fix(ai-help): refetch quota after an ai error

### PR DESCRIPTION
**Pull request by argl**
## Summary

(MP-708)

### Problem

If we get an error from the OpenAI API, we reinstate the user's quota server side and is not reflected correctly in the UI (input placeholder text)

### Solution

We trigger a refetch of the quota information on errors.

---

## How did you test this change?

Manually by running a local rumba server patched with triggering deliberate API errors.

### CODE REVIEW

File: client/src/plus/ai-help/use-ai.ts [hunk 1]
```diff
@@ -384,6 +384,7 @@ export function useAiChat({
       eventSourceRef.current?.close();
       eventSourceRef.current = undefined;
       setLoadingState("failed");
+      mutate("/api/v1/plus/ai/help/quota");
```
Comment (caugner) [inline client/src/plus/ai-help/use-ai.ts:387]: Nit: Shall we extract this constant here:

https://github.com/mdn/yari/blob/69395528f0943a99cc0fd033407b72830fb58f4f/client/src/plus/common/api.tsx#L1-L3

**Review by caugner — CHANGES_REQUESTED** [review]
LGTM, just one nit.

File: client/src/plus/ai-help/use-ai.ts (hunk 1 above)
Comment (argl) [reply client/src/plus/ai-help/use-ai.ts:387]: Good point!

**Review by caugner — APPROVED** [review]
LGTM, thanks!

**Pull request merged**